buildout.cfg

```diff
@@ -1,13 +1,16 @@
 [buildout]
-extends =
-	http://download.zope.org/zopetoolkit/index/1.0.8/zopeapp-versions.cfg
-    http://download.zope.org/Zope2/index/2.13.21/versions.cfg
-
-versions = versions
+index = https://pypi.org/simple/
```

Comment:
That's not needed, it is the default

Reply:
I don't understand the comment. In other plone package that I have I use a similar kind of index